utils/banks.js

```diff
@@ -236,7 +236,12 @@ export async function getDefaultFields(rating, bankname) {
     description4: '',
   }
   try {
-    const prismicData = await useBankPage(rating + 'bank')
+    var prismicData = await useBankPage(rating + 'bank')
+    if (rating === 'unknown') {
+      prismicData = await useBankPage('unknownbank-' + institutionType.toLowerCase().replace(' ', ''))
```

Comment:
if the new bank page isn't found, will it still default to the hardcoded default text? I think that's the intended behavior and that that's what it does, but wanted to make sure.

Reply:
yeah if there is no unkownbank prismic page for the type, it will use the hard coded text